new-lamassu-admin/src/pages/Transactions/DetailsCard.js

```diff
@@ -206,6 +259,22 @@ const DetailsRow = ({ it: tx }) => {
             errorElements
           )}
         </div>
+        <div>
+          {
+            <div
+              onClick={() => {
+                loadAndTxSummary(tx)
```

Comment:
This function name doesn't really tell me anything. Why not `downloadRawLogs`

Reply:
It was supposed to be loadTxSummary but your suggestion makes more sense